deploy/helm/trickster/templates/configmap.yaml

```diff
@@ -10,105 +10,191 @@ metadata:
 data:
   trickster.conf: |-
     [main]
-
-        # instance_id allows you to run multiple trickster processes on the same host and log to separate files
-        # Useful for baremetal, not so much for elastic deployments, so only uncomment if you really need it
-        #instance_id = 1
+    # instance_id allows you to run multiple trickster processes on the same host and log to separate files
+    # Useful for baremetal, not so much for elastic deployments, so only uncomment if you really need it
+    #instance_id = 1
 
     # Configuration options for the Proxy Server
     [proxy_server]
+    # listen_port defines the port on which Trickster's Proxy server listens.
+    # The default proxy is for Prometheus, so we use 9090 by default, just like Prometheus does
+    # listen_port = 9090
+    listen_port = {{ .Values.service.port }}
+    # listen_address defines the ip on which Trickster's Proxy server listens.
+    # empty by default, listening on all interfaces
+    # listen_address =
+    {{ if .Values.service.metricsAddress }}
+    listen_address = {{ .Values.service.address }}
```

Comment:
The naming of these two variables is confusing: `service.metricsAddress` and `service.address` -- I think you may have intended for them to be the same variable?

Reply:
There are two separate configurable values:
https://github.com/Comcast/trickster/blob/next/cmd/trickster/conf/example.conf#L13
https://github.com/Comcast/trickster/blob/next/cmd/trickster/conf/example.conf#L131

That are used differently in the code:
https://github.com/Comcast/trickster/blob/next/internal/config/config.go#L107
https://github.com/Comcast/trickster/blob/next/internal/config/config.go#L123

Reply:
This does remain true that these values are separate but this conversation is not relevant given our service discussion below.